index.html

```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
         <p class="contact-description">
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
-        <form class="contact-form">
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/xvolvllr"
+          method="POST"
+        >
           <input
             type="text"
             id="fullname"
             name="fullname"
             placeholder="Enter Full Name"
+            maxlength="30"
           />
```

Comment:
- [ ] A user cannot submit the form if one or more of the fields is empty, Please also add `required` attributeto the name field so the user can not submit the form if the name field is empty.